ibm_mq/tests/conftest.py

```diff
@@ -28,6 +28,12 @@ def instance():
     return inst
 
 
+@pytest.fixture
+def instance_metadata():
+    inst = copy.deepcopy(common.INSTANCE_METADATA)
+    return inst
```

Comment:
Is it enough to use existing `instance` fixture ? If so, we should avoid this new fixture IMO.